_includes/v21.2/ui/statement-details.md

```diff
@@ -0,0 +1,145 @@
+  - **Network usage** displays the amount of [data transferred over the network]({{ link_prefix }}architecture/reads-and-writes-overview.html) for statements with this fingerprint within the aggregation interval. If this value is 0, the statement was executed on a single node.
+  - **Max scratch disk usage** displays the maximum amount of data [spilled to temporary storage on disk]({{ link_prefix }}vectorized-execution.html#disk-spilling-operations) while executing statements with this fingerprint within the aggregation interval.
+
+**Statement details** displays information about the execution of the statement.
+
+- **Interval start time** represents the start time of the statistics aggregation interval for a statement. For example, if a statement is executed at 1:23PM it will fall in the 1:00PM - 2:00PM time interval.
+- **Nodes**: the nodes on which the statements executed. Click the node ID to view node statistics. <br><br>**Nodes** are not visible for {{ site.data.products.serverless }} clusters.
+- **Regions**: the regions on which the statements executed. <br><br>**Regions** are not visible for {{ site.data.products.serverless }} clusters.
+- **Database**: the database on which the statements executed.
+- **App**: the name specified by the [`application_name`]({{ link_prefix }}show-vars.html#supported-variables) session setting.
+- **Failed?**: whether the statement failed to execute.
+- **Used cost-based optimizer?**: whether the execution used the [cost-based optimizer]({{ link_prefix }}cost-based-optimizer.html).
+- **Distributed execution?**: whether the execution was distributed.
+- **Vectorized execution?**: whether the execution used the [vectorized execution engine]({{ link_prefix }}vectorized-execution.html).
+- **Transaction type**: the type of transaction (implicit or explicit).
+- **Last execution time**: when the statement was last executed.
+
+**Execution counts** displays execution statistics for the SQL statement fingerprint.
+
+  - **First attempts**: the cumulative number of first attempts at executing statements with this fingerprint within the aggregation interval.
+  - **Total executions**: the total number of executions of statements with this fingerprint. It is calculated as the sum of first attempts and retries.
+  - **Retries**: the cumulative number of [retries]({{ link_prefix }}transactions.html#transaction-retries) of statements with this fingerprint within the aggregation interval.
+  - **Max Retries**: the highest number of retries of a single statement with this fingerprint within the aggregation interval. For example, if three statements with the same fingerprint had to be retried 0, 1, and 5 times, then the Max Retries value for the fingerprint is 5.
+
+### Diagnostics
+
+The **Diagnostics** section allows you to activate and download diagnostics for the SQL statement fingerprint.
+
+{{site.data.alerts.callout_info}}
+The **Diagnostics** tab is not enabled for {{ site.data.products.serverless }} clusters.
```

Comment:
should we also mention this tab is also not enabled if user has `VIEWACTIVITYREDACTED` role?

Reply:
We do have the warning at the bottom of this section: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/v22.1/ui-statements-page.html#diagnostics that says you can't download bundles.  Somehow I missed the tab behavior. I'll fix that. Thanks for the catch.